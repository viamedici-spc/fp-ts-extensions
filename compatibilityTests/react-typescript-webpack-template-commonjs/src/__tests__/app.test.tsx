import { render } from '@testing-library/react';

import App from '../app';

test('renders app component', () => {
  const { container, getByRole } = render(<App />);

  expect(getByRole('main')).toBeInTheDocument();
  expect(container).toMatchSnapshot();
});
