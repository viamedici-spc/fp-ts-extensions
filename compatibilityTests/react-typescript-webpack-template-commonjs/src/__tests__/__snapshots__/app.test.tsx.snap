// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders app component 1`] = `
<div>
  <div
    class="app"
    role="main"
  >
    <header
      class="app-header"
    >
      <img
        alt="logo"
        class="app-logo"
        src="test-file-stub"
      />
      <p>
        Edit 
        <code>
          src/app.tsx
        </code>
         and save to reload.
      </p>
      <a
        class="app-link"
        href="https://reactjs.org"
        rel="noopener noreferrer"
        target="_blank"
      >
        Learn React
      </a>
    </header>
  </div>
</div>
`;
